src/module/mod.rs

```diff
@@ -132,24 +133,23 @@ impl Module {
         let mut ret = Module::default();
         ret.config = config.clone();
         let mut indices = IndicesToIds::default();
-        let mut validator = Validator::new();
-        validator.wasm_features(WasmFeatures {
-            reference_types: !config.only_stable_features,
-            multi_value: true,
-            bulk_memory: !config.only_stable_features,
-            simd: !config.only_stable_features,
-            threads: !config.only_stable_features,
-            multi_memory: !config.only_stable_features,
-            ..WasmFeatures::default()
-        });
+
+        // TODO: how should we handle config.only_stable_features?
+        let wasm_features = WasmFeatures::default();
```

Comment:
For `only_stable`, it seems fine to just treat that as the default. That is, if `only_stable_features` is false we can enable all experimental features.